install/updates/froxlor/0.9/update_0.9.inc.php

```diff
@@ -3513,3 +3513,13 @@
 	showUpdateStep("Updating from 0.9.38-rc1 to 0.9.38-rc2", false);
 	updateToVersion('0.9.38-rc2');
 }
+/*
+if (isDatabaseVersion('201610070')) {
```

Comment:
Update looks correct, but when "active" use the current date and not 201610270 as new dbversion :)